docs/output.md

```diff
@@ -1,35 +1,71 @@
 # nf-core/cutandrun: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/cutandrun/output](https://nf-co.re/cutandrun/output)
-
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
-
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline. Example plots are taken from the pdf report which details summary details and analyses specific to CUT&Run/CUT&Tag data, and the MultiQC report, which summarises results from some tools used, at the end of the pipeline.
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-* [FastQC](#fastqc) - Raw read QC
-* [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
-* [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+- [nf-core/cutandrun: Output](#nf-corecutandrun-output)
+    - [Introduction](#introduction)
+    - [Pipeline overview](#pipeline-overview)
+    - [Preprocessing](#preprocessing)
+        - [Samplesheet check](#samplesheet-check)
+        - [Fastq merging](#fastq-merging)
+        - [FastQC](#fastqc)
+        - [TrimGalore](#trimgalore)
+    - [Alignment](#alignment)
+        - [Bowtie 2](#bowtie-2)
+    - [Alignment post-processing](#alignment-post-processing)
+        - [samtools](#samtools)
+        - [picard MarkDuplicates/RemoveDuplicates](#picard-markduplicatesremoveduplicates)
+    - [Peak Calling](#peak-calling)
+        - [Bam to bedgraph](#bam-to-bedgraph)
+        - [Clip bedfiles](#clip-bedfiles)
+        - [Bed to bigwig](#bed-to-bigwig)
+        - [SEACR peak calling](#seacr-peak-calling)
+        - [BEDtools](#bedtools)
+    - [Reporting](#reporting)
+        - [Python reporting](#python-reporting)
+        - [MultiQC](#multiqc)
+        - [IGV](#igv)
+        - [Deeptools](#deeptools)
+    - [Workflow reporting and genomes](#workflow-reporting-and-genomes)
+        - [Reference genome files](#reference-genome-files)
+        - [Pipeline information](#pipeline-information)
```

Comment:
No need for a markdown based ToC anymore. It will be generated from the headings on the website automatically